File: iOS/iOS-MVVM/iOS-MVVM/Models/Crew.swift
//
//  Crew.swift
//  iOS-MVVM
//
//  Created by Guillian on 2019-04-16.
//  Copyright © 2019 AffinityClick. All rights reserved.
//

import Foundation

struct Crew: Decodable {
    let creditId: String
    let department: String
    let gender: Int
    let id: Int
    let name: String
    let profilePath: String
}

File: android/MVVM/app/src/main/java/com/affinityclick/mvvm/network/models/Crew.kt
package com.affinityclick.mvvm.network.models

import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Crew(
  @SerializedName("credit_id") @Expose var creditId: String? = null,
  @SerializedName("department") @Expose var department: String? = null,
  @SerializedName("gender") @Expose var gender: Int? = null,
  @SerializedName("id") @Expose var id: Int? = null,
  @SerializedName("job") @Expose var job: String? = null,
  @SerializedName("name") @Expose var name: String? = null,
  @SerializedName("profile_path") @Expose var profilePath: String? = null,
) : Parcelable